_posts/2015-03-stitch-fix-heart-unix.md

```diff
@@ -0,0 +1,682 @@
+---
+title: Stitch Fix &#10084; UNIX
+layout: posts
+author: Simeon Willbanks
+author_url: 'https://twitter.com/simeonwillbanks'
+published: true
+location: "Everywhere"
```

Comment:
@ebressert @davetron5000 Collaboratively written across the continent, so `location: "Everywhere"`.